moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -168,7 +172,8 @@ const ProjectComponentEdit = ({
   const { saveActionState, saveActionDispatch } = useSaveActionReducer();
 
   /**
-   * Generates an initial list of component types, subtypes and counts
+   * Generates an initial list of component types, subtypes and counts (counts is total number of subtypes)
+   * todo chia: ask if this initial list ever changes? or is this a constant list of component types, subtypes
    */
   const initialTypeCounts = data // Do we have data?
```

Comment:
does intialTypeCounts change during mapping, or is it a static array? I think its a constant list, but I wasnt sure if initial meant that it might change along the way?

Reply:
@chiaberry you are correct. it's static.